documentation/modules/post/linux/gather/haserl_read.md

```diff
@@ -0,0 +1,59 @@
+## Vulnerable Application
+
+[Haserl](http://haserl.sourceforge.net/) is an unmaintained tool to use LUA as CGI in web servers.
+On Linux, if haserl is usually suid root, sot that it will attempt to drop its privilege to the uid/gid of the owner of the cgi script,
+similar to suexec in Apache.
+
+Haserl could have been a thing of the past, but it's used in Alpine Linux'
+[Alpine Configuration
+Framework](https://wiki.alpinelinux.org/wiki/Alpine_Configuration_Framework_Design),
+which is commonly used on this distribution.
+
+This module exploits the fact that calling haserl on a file will make it not only not drop its priviledges,
```

Comment:
In this case, it's not really dropping privilges, but increasing privileges. So perhaps:

```suggestion
This module exploits the fact that calling haserl on a file will make it not only change the effective UID,
```